### PR TITLE
Multiple Fix & improvement following the server migration

### PR DESCRIPTION
**Pull request by victor-paumier**
- fix nbBuddies on remove waiting confirm buddy
- do not remove archived buddies
- show activated section on list
- add CA-SERVER in env.dist
- fix section warning on change semester 
- improve edit user
- improve cron warning mentors
- improve deployer

### CODE REVIEW

File: src/Repository/BuddyRepository.php [hunk 1]
```diff
@@ -93,15 +93,21 @@ public function getMyBuddiesArchived(User $user)
...
-            ->where('umentor.archive = 0')
-            ->andWhere('umentee.archive = 0')
+            ->where('umentor.archive = :archive')
+            ->andWhere('umentee.archive = :archive')
+            ->setParameter('archive', false)
         ;
 
+        if (!$includeArchived) {
+            $qb
+                ->andWhere('b.archive = :archive');
```
Comment (JeremieSamson) [inline src/Repository/BuddyRepository.php]: better `$qb->andWhere('b.archive = :archive');` no ?

File: src/Repository/BuddyRepository.php [hunk 2]
```diff
@@ -116,12 +122,15 @@ public function getBuddiesToRemove()
         $qb = $this->createQueryBuilder('b');
 
         $qb->innerJoin('b.mentor', 'mentor')
```
Comment (JeremieSamson) [inline src/Repository/BuddyRepository.php]:
```suggestion
        $qb
            ->innerJoin('b.mentor', 'mentor')
```

File: src/Repository/BuddyRepository.php [hunk 3]
```diff
@@ -131,12 +140,15 @@ public function getBuddiesToWarn()
     {
         $qb = $this->createQueryBuilder('b');
         $qb->innerJoin('b.mentor', 'mentor')
```
Comment (JeremieSamson) [inline src/Repository/BuddyRepository.php]:
```suggestion
        $qb
            ->innerJoin('b.mentor', 'mentor')
```

File: src/Repository/BuddyRepository.php (hunk 1 above)
Comment (victor-paumier) [reply src/Repository/BuddyRepository.php]: done

File: src/Repository/BuddyRepository.php (hunk 2 above)
Comment (victor-paumier) [reply src/Repository/BuddyRepository.php]: done

File: src/Repository/BuddyRepository.php (hunk 3 above)
Comment (victor-paumier) [reply src/Repository/BuddyRepository.php]: done

**Pull request merged**